### PR TITLE
incorporating values

### PR DESCRIPTION
**Pull request by cpolitano**
- bring more hospitality into Intro
- small text edits to Unacceptable Behavior
- generalized Reporting a Problem to refer to the People team, not any one individual

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +16,7 @@ The contents of this code of conduct apply to our interactions in various areas
 
 We want our team to be fun, productive, and safe for all members. We've adopted the Social Tables [core values](https://www.socialtables.com/company/about/#values), in ways that specifically apply to the craft of Software Engineering at Social Tables:
 
-**We recognize that "It Takes a Village" as we work alongside other teams.** Customer Success, Sales, Marketing, and Operations team members and others across the company are our peers and collaborators, not our clients. We maintain open lines of communication and foster good working relationships—indeed, friendships—with our colleagues. That collaboration is core to our success.
+**We recognize that "It Takes a Village" as we work alongside other teams.** Customer Success, Sales, Marketing, and Operations team members and others across the company are our peers and collaborators, not our clients. We maintain open lines of communication and foster good working relationships—-indeed, friendships—-with our colleagues. That collaboration is core to our success.
```
Comment (danmactough) [inline README.md]: @cpolitano Did you mean to change this to an m-dash followed by a hyphen? I think we need to just do standardization pass on things like curly quotes, m-dash vs. two hyphens, etc.

File: README.md [hunk 2]
```diff
@@ -46,12 +48,12 @@ We're creating a lasting team built on teamwork and collaboration. [It takes a v
...
 
-We have two-way, transparent, and ethical conversations. [Always honest](https://). Any behavior or language that is unwelcoming is strongly discouraged. Such exclusionary behavior can take many forms, including [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—subtle put-downs, whether deliberate or unconscious. Regardless of intent, microaggressions can have a significant negative impact on victims and have no place on our team.
+We have two-way, transparent, and ethical conversations. [Always honest](https://). Any behavior or language that is unwelcoming is strongly discouraged. Such exclusionary behavior can take many forms, including [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—-subtle put—-downs, whether deliberate or unconscious. Regardless of intent, microaggressions can have a significant negative impact on colleagues and have no place on our team.
```
Comment (danmactough) [inline README.md]: 💯 for removing "victims". That extra m-dash in the middle of "put-downs" we don't want.

File: README.md [hunk 3]
```diff
@@ -46,12 +48,12 @@ We're creating a lasting team built on teamwork and collaboration. [It takes a v
...
 
-There are a host of behaviors and language common on tech teams which are worth noting as specifically unwelcome:
+Here is some guidance on how to avoid certain behaviors and language which are specifically unwelcome:
```
Comment (danmactough) [inline README.md]: `s/is some guidance on/are some suggestions for/`?

File: README.md [hunk 4]
```diff
@@ -60,29 +62,29 @@ There are a host of behaviors and language common on tech teams which are worth
...
 
 3. **Talk to your manager.** Your manager probably knows quite a lot about the dynamics of your team, which makes them a good person to look to for advice. They may also be able to talk directly to the colleague in question if you feel uncomfortable or unsafe doing so yourself. Finally, your manager will be able to help you figure out how to ensure that any conflict with a colleague doesn’t interfere with your work.
 
-4. **Talk to Sam.** Sam is happy to talk to you about the problem and help figure out what steps to take. Sam is good at listening to concerns about small violations, but she’ll also be able to help out in situations where more drastic action needs to be taken.
+4. **Talk to People.** The People team is happy to talk to you about the problem and help figure out what steps to take. They are good at listening to concerns about small violations, but will also be able to help out in situations where more drastic action needs to be taken.
 
-## Committing to improvement
+## Committed to Quality
```
Comment (danmactough) [inline README.md:81]: nice 😎

**Comment by cpolitano** [comment]
thanks @danmactough, all double-hyphens for now.

**Comment by danmactough** [comment]
> for now

oh, man. i can't wait for what's next!

**Pull request merged**